src/app/screens/Send/index.tsx

```diff
@@ -187,6 +187,7 @@ function Send() {
                   <QrCodeIcon className="h-6 w-6 text-blue-500" />
                 </button>
               }
+              autoFocus
```

Comment:
can we move this one up? the `onChange` and `endAdornment` are super long and this might be missed.
maybe above onchange?